config/local/generate_toml.go

```diff
@@ -53,24 +54,18 @@ var LocalConfig = config.OracleConfig{
 	},
 	UpdateInterval: 1 * time.Second,
 	Providers: []config.ProviderConfig{
+		// -----------------------------------------------------------	//
+		// ---------------------Start API Providers--------------------	//
+		// -----------------------------------------------------------	//
+		//
+		// NOTE: Some of the provider's are only capable of fetching data for a subset of
+		// all of the currency pairs. Before adding a new market to the oracle, ensure that
+		// the provider supports fetching data for the currency pair.
 		{
 			// -----------------------------------------------------------	//
-			// ---------------------Start API Providers--------------------	//
-			// -----------------------------------------------------------	//
-			//
-			// NOTE: Some of the provider's are only capable of fetching data for a subset of
-			// all of the currency pairs. Before adding a new market to the oracle, ensure that
-			// the provider supports fetching data for the currency pair.
+			// ---------------------Start Binance API----------------------	//
 			Name: binance.Name,
-			API: config.APIConfig{
-				Name:       binance.Name,
-				Atomic:     true,
-				Enabled:    true,
-				Timeout:    500 * time.Millisecond,
-				Interval:   1 * time.Second,
-				MaxQueries: 1,
-				URL:        binance.US_URL,
-			},
```

Comment:
I think at some point we will want to fine tune these so i moved all of the websocket/api configs to util files where we can centrally have default configs.